.github/ISSUE_TEMPLATE/feature-request.yml

```
@@ -0,0 +1,41 @@
+name: 🚀 | Feature request
+description: >
+  Request a new feature
+title: "[🚀 | Feature request]: "
+labels:
+  - triage
+body:
+  - type: dropdown
+    id: related-to-problem
+    attributes:
+      label: Is your feature request related to a problem?
+      multiple: false
+      options:
+        - "Yes"
+        - "No"
+    validations:
+      required: true
+  - type: textarea
+    id: problem-description
+    attributes:
+      label: If your feature request is related to a problem describe it
+      description: |
+        Describe the problem you are trying to solve with this feature request.
```

I'd move the "why it is important for you" here because is related to the problem more than to the solution.
```suggestion
      label: What you would like to accomplish
      description: |
        Describe the problem you are trying to solve with this feature request,
        why it is needed and why it is important for you.
```